bookdown/03-optimization-hyperband.Rmd

```diff
@@ -183,26 +183,26 @@ But it is also possible to define a custom `r ref("Sampler")` for each hyperpara
 library(mlr3learners)
 set.seed(123)
 
-params = list(
-  ParamInt$new("nrounds", lower = 1, upper = 16, tag = "budget"),
-  ParamDbl$new("eta",     lower = 0, upper = 1),
-  ParamFct$new("booster", levels = c("gbtree", "gblinear", "dart"))
+params = ps(
+  nrounds = p_int(lower = 1, upper = 16, tags = "budget"),
+  eta = p_dbl(lower = 0, upper = 1),
+  booster = p_fct(levels = c("gbtree", "gblinear", "dart"))
 )
 
 inst = TuningInstanceSingleCrit$new(
   tsk("iris"),
   lrn("classif.xgboost"),
   rsmp("holdout"),
   msr("classif.ce"),
-  ParamSet$new(params),
+  params,
   trm("none") # hyperband terminates itself
 )
 
 # beta distribution with alpha = 2 and beta = 5
 # categorical distribution with custom probabilities
 sampler = SamplerJointIndep$new(list(
-  Sampler1DRfun$new(params[[2]], function(n) rbeta(n, 2, 5)),
-  Sampler1DCateg$new(params[[3]], prob = c(0.2, 0.3, 0.5))
+  Sampler1DRfun$new(params$params$eta, function(n) rbeta(n, 2, 5)),
+  Sampler1DCateg$new(params$params$booster, prob = c(0.2, 0.3, 0.5))
```

Comment:
Could you call `params` `ps` instead? it becomes confusing here.

Reply:
you mean the variable params? I can name it `tune_ps`, calling it `ps` would be confusing because of the `ps()` function we now have...

Reply:
`tune_ps` sounds perfect.

Reply:
:+1: